include/api/CAnomalyJobConfig.h

```diff
@@ -9,6 +9,7 @@
 #include <core/CLogger.h>
 
 #include <api/CDetectionRulesJsonParser.h>
+#include <api/CFieldConfig.h>
```

Comment:
This addition won't be required after switching to `CDetectionRulesJsonParser::TStrPatternSetUMap`.